src/app/forms/templates/general/approval-type.template.ts

```diff
@@ -15,9 +15,7 @@ export function getTypeApprovalSection(vehicleType: VehicleTypes): FormNode {
         type: FormNodeTypes.CONTROL,
         editType: FormNodeEditTypes.SELECT,
         options: getOptionsFromEnum(approvalType),
-        validators: [
-          { name: ValidatorNames.Required }
-        ],
+        validators: vehicleType === VehicleTypes.PSV ? [] : [{ name: ValidatorNames.Required }],
```

Comment:
Believe this is mandatory for all vehicle types https://dvsa.atlassian.net/wiki/spaces/HVT/pages/49679745/Amend+Technical+Record

Reply:
👍

Reply:
fixt